examples/weather_app/src/modules/weather/weather_controller.rs

```diff
@@ -21,9 +21,11 @@ impl WeatherController {
     #[get("/weather/<location>/<city>")]
     #[check(WeatherGate)]
     async fn get_location(&self, params: Param) -> String {
-        self.weather_service
-            .get_location_weather(params.get("location").unwrap().as_str())
-            .await
+        let location = params.get("location").unwrap_or_default();
+        if location.is_empty() {
+            eject!("Location is required");
```

Comment:
This is how `eject!` would be used ideally. However, the current WIP doesn't change the status code. It only updates the response body